src/Generator.php

```diff
@@ -91,6 +91,11 @@ class Generator
      */
     protected $fileSystem;
 
+    /**
+     * @var array
+     */
+    protected $config;
```

Comment:
Can we be more precise and name this something like `documentationConfig`.

On the other hand - in the factory now we destruct whole `documentationConfig` and pass it's parts to generator as a separate params. This prop will hold everything now, so we will have duplication... 🤔